internal/provider/resource_tfe_variable_test.go

```diff
@@ -192,6 +194,38 @@ func TestAccTFEVariable_update_key_sensitive(t *testing.T) {
 	})
 }
 
+func TestAccTFEVariable_valueWriteOnly(t *testing.T) {
+	variable := &tfe.Variable{}
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+
+	variableValue1 := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+	variableValue2 := variableValue1 + 42
+
+	resource.Test(t, resource.TestCase{
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.SkipBelow(version.Must(version.NewVersion("1.11.0"))),
+		},
+		ProtoV5ProviderFactories: testAccMuxedProviders,
+		CheckDestroy:             testAccCheckTFEVariableDestroy,
+		Steps: []resource.TestStep{
```

Comment:
Consider including a test for the `ConflictsWith` validator?

We can also use `resource.TestCheckNoResourceAttr` to assert that there's no value in state.